src/printer.js

```diff
@@ -2435,7 +2451,9 @@ function printAstToDoc(ast, options) {
     );
   }
 
-  return printGenerically(FastPath.from(ast));
+  const doc = printGenerically(FastPath.from(ast));
+  docUtils.propagateBreaks(doc);
```

Comment:
I'm not sure if we want to do this here or in `doc-printer.js`. I did it here because the doc debugging in the browser would show the changes.